dist/scripts/api-console.js

```diff
@@ -2118,7 +2118,10 @@
     };
 
     $rootScope.$on(PARSE_SUCCESS, function(e, raml) {
-      setPromise($q.when(raml));
+      // Only act when raml is valid
+      if (raml.title) {
```

Comment:
In the new parser, title is a function, so if there is any valid RAML (even one with an empty title) this `if` condition will be true. There is one case in designer where an empty raml object is sent in a parse success event. This change is to avoid errors in that situation.